wrangler-core/src/main/java/io/cdap/wrangler/parser/GrammarBasedParser.java

```diff
@@ -77,7 +77,10 @@ public GrammarBasedParser(String namespace, String recipe, DirectiveRegistry reg
       CompileStatus status = compiler.compile(recipe);
       if (!status.isSuccess()) {
         Iterator<SyntaxError> errors = status.getErrors();
-        throw new DirectiveParseException(errors.next().getMessage(), errors);
+        String prefix = "Encountered syntax error, please ensure that all directives are valid "
+            + "(list of valid directives can be found at "
+            + "https://github.com/data-integrations/wrangler/blob/develop/README.md):\n";
+        throw new DirectiveParseException(prefix + errors.next().getMessage(), errors);
```

Comment:
how does multi line message look on the UI now?

Reply:
it looks the same in the new logs viewer: 
![image](https://user-images.githubusercontent.com/54917202/92953319-5d4ce600-f416-11ea-96af-32d985abf29f.png)

Reply:
would this also get propagated on commandline?

Reply:
It doesnt seem to: 
![image](https://user-images.githubusercontent.com/54917202/92956951-9be59f00-f41c-11ea-986c-d18b80b75644.png)

Reply:
Correction: It does propagate to commandline, I didnt copy the correct artifacts over 

![image](https://user-images.githubusercontent.com/54917202/92970468-92b3fc80-f433-11ea-9bb4-ee8d6eb00bc9.png)